config/urls.py

```diff
@@ -15,14 +15,22 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 
+from debug_toolbar.toolbar import debug_toolbar_urls
 from django.conf import settings
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import include, path
 
-urlpatterns = [
-    path("", include(("app.main.urls", "main"), namespace="main")),
-    path("healthcheck/", include("app.healthcheck.urls")),
-    path("search/", include(("app.search.urls", "search"), namespace="search")),
-    path("admin/", admin.site.urls),
-] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+urlpatterns = (
+    [
+        path("", include(("app.main.urls", "main"), namespace="main")),
+        path("healthcheck/", include("app.healthcheck.urls")),
+        path(
+            "search/",
+            include(("app.search.urls", "search"), namespace="search"),
+        ),
+        path("admin/", admin.site.urls),
+    ]
+    + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    + debug_toolbar_urls()
```

Comment:
Could this `debug_toolbar_urls()` fail  for a non root user? 

Could this be added only if installed? as in 

`from django.apps import apps`

```
if apps.is_installed("debug_toolbar"):
    urlpatterns = [
        path("__debug__/", include("debug_toolbar.urls")),
    ] + urlpatterns
```

Reply:
This would be good. Please feel free to edit.